inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -343,7 +380,7 @@ lithium_electrolyte-reactions:
 electrolyte-reactions:
 - equation: Li+[elyt] + O2(e) + electron <=> LiO2[elyte]  # Reaction 2
```

Comment:
Update species names here to match above comment.

Reply:
I will fix this after we clear up the confusion between the cathode and carbon phase which I think I'm still confused about....